refs/wg21.json

```diff
@@ -78374,7 +78374,7 @@
         "href": "https://wg21.link/lwg2304",
         "status": "C++14",
         "authors": [
-            "Joaquín M López Muñoz"
+            "Joaquín Mª López Muñoz"
```

Comment:
Various names suffered from varying degrees of mojibake. When a name appeared both with fancy characters and either without or baked, I corrected to the fancy flavor.

My tooling likes to use lowercase ascii letters for commit messages, but those are really just for me to maintain an ordered sequence of tasks, I don't expect the commit messages to be retained when merged.